### PR TITLE
Hotfix: asterisk field

### PR DESCRIPTION
**Pull request by jonnybazookatone**
How to behave if someone passes fl=*.

### CODE REVIEW

File: solr/views.py [hunk 1]
```diff
@@ -70,6 +70,8 @@ def cleanup_solr_request(payload):
                 fields = filter(lambda x: x not in disallowed, fields)
             if len(fields) == 0:
                 fields.append('id')
+            if '*' in fields:
+                fields = current_app.config.get('SOLR_SERVICE_ALLOWED_FIELDS')
             payload['fl'][0] = ','.join(fields)
```
Comment (romanchyla) [inline solr/views.py]: wow, i didn't know '*' was valid. But there are two more edge cases to take care of. Could you pls change also two other lines?

```fields = [x.strip() for x in fields[0].split(',')]```

and 

```payload['fl'] = ','.join(fileds)```

just to be sure

File: solr/tests/unittests/test_solr.py [hunk 2]
```diff
@@ -11,6 +11,24 @@
...
+        return a
+
+    def test_cleanup_solr_request(self):
+        """
+        Simple test of the cleanup classmethod
+        """
+        payload = {'fl': ['*,bibcode,title']}
+        cleaned = SolrInterface.cleanup_solr_request(payload)
+
+        self.assertNotIn('*', cleaned['fl'][0])
```
Comment (romanchyla) [inline solr/tests/unittests/test_solr.py]: this test succeeds but is incorrect - i'll fix it after the merge
